.circleci/config.yml

```diff
@@ -20,18 +20,18 @@ jobs:
         # This branch if available
         - v1-dep-{{ .Branch }}-
         # Default branch if not
-        - v1-dep-master-
+        # - v1-dep-master-
```

Comment:
Why disabling these?

Reply:
So that it never grabs the cache from `master`, etc.
If it does, we get version conflicts with `hawthorn`.

Reply:
ok, cool.